content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -16,6 +16,7 @@ This page lists some of the available add-ons and links to their respective inst
 ## Networking and Network Policy
 
 * [ACI](https://www.github.com/noironetworks/aci-containers) provides integrated container networking and network security with Cisco ACI.
+* [Antrea](https://github.com/vmware-tanzu/antrea) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane.
```

Comment:
```suggestion
* [Antrea](https://antrea.io/) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane.
```
We prefer to link to docs ahead of source code

Reply:
sure, will update it. thanks!

Reply:
Hi @sftim , I just updated it, Could you help to review it again? thanks!